modules/nw-rfhe-troubleshooting.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
```

Comment:
Remove the file from the PR if it is not being used.

Reply:
Done.